src/style.css

```diff
@@ -0,0 +1,70 @@
+@font-face {
+  font-family: Avenir;
+  src: url(Avenir.ttc);
+  font-weight: 300;
+}
+  
+.search-block {
+  width: 100%;
+  box-sizing: border-box;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  outline: none;
+  color: #3d4e61;
+  font-family: Avenir;
+}
+  
+.search-block:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+  
+.search-block:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border-radius: 3px;
+  font-weight: 700;
+}
+  
+.big-search {
+  height: 70px;
+  padding-left: 62px;
+  font-size: 16px;
+  line-height: 22px;
+}
+  
+.big-search-container {
+  position: relative;
+  margin: 20px 0;
+}
+  
+.big-search-container::before {
+  content: "";
+  position: absolute;
+  background-image: url(images/Search.svg);
+  width: 19px;
+  height: 19px;
+  margin-top: 25px;
+  margin-left: 26px;
+}
+  
+.small-search {
+  height: 42px;
+  padding-left: 33px;
+  font-size: 14px;
+  line-height: 16px;
```

Comment:
We don't have any basic search bar, so why we should use there big and small search class names? We have search-input and search-input - small (--small). Therefore, we set all styles that are supposed to be for the "large" input for all inputs in general, and with the modifier small we change only the necessary values to make our input smaller.